src/modules/main-nav/MainNav.scss

```diff
@@ -1,4 +1,200 @@
+	height: 100vh;
+}
+
+.main-nav-container{
+	width: 100%;
+	height: 20%;
+	background-color: white;
+	display: flex;
+	align-content: center;
+	justify-content: center;
+
+	&__button{
+		display: flex;
+		align-items: center;
+		width: 20%;
+		height: 100%;
+		padding: 0 20px;
+		position: relative;
+
+		@include large-screen{
+			display: none;
+		}
+	}
+
+	.button-closed {
+		display: block;
+		width: 40px;
+		height: 40px;
+		border-radius: 2px;
+		background: darken(white, 5%);
+		border: 1px solid darken(green, 15%);
+
+		&-line {
+			display: block;
+			height: 2px;
+			width: 100%;
+			background: black;		
+		
+			&::before,
+			&::after {
+				display: block;
+				content: '';
+				height: 2px;
+				width: 100%;
+				background: black;
+				position: relative;
+			}
+		
+			&::before {
+				top: -6px;
+			}
+		
+			&::after {
+				bottom: -4px;
+			}
+		}
+
+		&-cross{
+			position: relative;
+  			display: inline-block;
+  			width: 30px;
+  			height: 30px;
+  			overflow: hidden;
+
+  			&::before, &::after {
+    			content: '';
+    			position: absolute;
+    			height: 2px;
+    			width: 100%;
+    			top: 50%;
+    			left: 0;
+    			margin-top: -2px;
+    			background: #000;
+  			}
+
+  			&::before {
+    			transform:rotate(45deg);
+  			}
+  			
+  			&::after {
+    			transform:rotate(-45deg);
+  			}
+		}	
+	}
+
+	.main-nav-container__logo{
+		width: 80%;
+		height: 100%;
+
+		.logo{
+			width: 20vh;
+			height: 90%;
+		}
+	}
+
+}
+
+.container{
+	position: relative;
+	width: 100%;
+	height: 80%;
+
+	@include large-screen{
+		position: absolute;
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+	}
+
+	.main-nav-menu{
+		position: absolute;
+		background-color: green;
+		width: 100%;
+		height: 100%;
+
+		&__item{
+			width: 100%;
+			height: 40px;
+			justify-content: center;
+			display: inline-block;
+			text-align: center;
+			align-items: center;
+
+			@include large-screen{
+				border: none;
+				z-index: 1;
+			}
+
+		}
+
+		&--opened{
+			z-index: 1;
+		}
+
+		@include large-screen{
+			position: relative;
+			display: flex;
+			flex-direction: row;
+			justify-content: center;
+			align-items: center;
+			height: 40px;	
+			background-color: white;
+		}
+
+	}
+
+	.article{
+		position: absolute;
+		background-color: blue;
+		width: 100%;
+		height: 100%;
+
+		@include large-screen{
+			position: relative;
+		}
+	}
+
+}
+ 
+button{
+	background:none;
+	border:none;
+	font-size:1em;
+	color: black;
+}
+
+.item-dropdown{
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+
+	&__arrow {
+		border-left: 5px solid transparent;
+		border-right: 5px solid transparent;
+		border-top: 5px solid #000;
+	}
+
+	&__menu {
+		width: 100%;
+		display: block;
+		text-align: center;
+		visibility: hidden;
+	}
+
+	&--opened a {
+		display: block;
+		visibility: visible;
+
+		@include large-screen{
+			background-color: #f5f5f5;
+			min-width: 160px;
+		}
+	}
 }
```

Comment:
Instead of writing a bunch of comments in this file, I would suggest a few things that you should check in general:
- BEM usage. Is it being used everywhere? Is it being used correctly?
- Does this file contains code that doesn't necessarily belongs to the MainNav?
- Am I using selector including tags? If the answer is yes, please avoid that.